packages/types/src/globals/WildcardPathFlags.ts

```diff
@@ -31,9 +31,9 @@ export const WildcardPathFlags = {
     wildcardSkipAttributeList: BitFlag(2),
 
     /**
-     * Reserved
+     * Skip the EventList global attribute during wildcard expansion.
```

Comment:
Please revert this rename because should be that way since we official do 1.4 ... but we removed events already ... so yes it is a "manual ignreo change"

Reply:
Ok I do an override in another PR ... all good